e2e/utils.go

```diff
@@ -1207,3 +1211,53 @@ func addTopologyDomainsToDSYaml(template, labels string) string {
 	return strings.ReplaceAll(template, "# - \"--domainlabels=failure-domain/region,failure-domain/zone\"",
 		"- \"--domainlabels="+labels+"\"")
 }
+
+// createCustomConfigMap provides multiple clusters information.
+func createCustomConfigMap(pluginPath string, c kubernetes.Interface, getSubVolGrp map[string]string) {
```

Comment:
change getSubVolGrp to clusterInfo?

Reply:
clusterInfo exists as a struct and is used in the same program which might add confusion https://github.com/ceph/ceph-csi/blob/master/e2e/utils.go#L345 ... Maybe `subvolgrpInfo` or any other name suggestion?

Reply:
Yeah anything will be better with meaningful name

Reply:
Okay, Thanks :)